src/javax/obex/Authenticator.java

```diff
@@ -0,0 +1,27 @@
+/*
+	This file is part of FreeJ2ME.
+
+	FreeJ2ME is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	FreeJ2ME is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with FreeJ2ME.  If not, see http://www.gnu.org/licenses/
+*/
+package javax.obex; 
+
+public interface Authenticator
+{
+
+    PasswordAuthentication onAuthenticationChallenge(String description, 
+    boolean isUserIdRequired, boolean isFullAccess);
```

Comment:
Why split this across two lines?

Reply:
I was trying to make the lines shorter, kinda like it's done in older C standards. Maybe it isn't useful here, so i'll get those back to one longer line.